lua/nvim-treesitter-textobjects/shared.lua

```diff
@@ -75,20 +76,26 @@ local get_query_matches = memoize(function(bufnr, query_group, root, root_lang)
   local matches = {}
   local start_row, _, end_row, _ = root:range()
   -- The end row is exclusive so we need to add 1 to it.
-  for pattern, match, metadata in query:iter_matches(root, bufnr, start_row, end_row + 1) do
+  for pattern, match, metadata in query:iter_matches(root, bufnr, start_row, end_row + 1, { all = true }) do
```

Comment:
That will no longer be necessary in the near future; we should target nightly on this branch.

Reply:
Sure, I'll remove it and hopefully we can merge after https://github.com/neovim/neovim/pull/30193 lands?

Reply:
Yeah, it won't take long. I can take a stab at docs tomorrow; in any case, that shouldn't block it so I might just YOLO merge.

Reply:
Nevermind I read your prior comment

Reply:
Neovim PR is merged; feel free to remove now (need to wait till tomorrow for nightlies -- and thus CI -- to have that, but we don't necessarily need to care about this on `main`).